Sources/KSCrashRecordingCore/KSDynamicLinker.c

```diff
@@ -84,16 +135,17 @@ static uintptr_t firstCmdAfterHeader(const struct mach_header *const header)
  * @param address The address to examine.
  * @return The index of the image it is part of, or UINT_MAX if none was found.
  */
-static uint32_t imageIndexContainingAddress(const uintptr_t address)
+static const struct mach_header *imageContainingAddress(const uintptr_t address, char **outName)
 {
-    const uint32_t imageCount = ksbic_imageCount();
-    const struct mach_header *header = 0;
+    uint32_t count = 0;
+    const struct dyld_image_info *images = ksbic_beginImageAccess(&count);
```

Comment:
This should probably have a NULL guard, I think?

Or can we guarantee that `count` will always be 0 on return when we return a NULL (regardless of whether the caller initialized it or not)?